client/src/router.jsx

```diff
@@ -0,0 +1,23 @@
+import {
+  createBrowserRouter,
+} from 'react-router-dom';
+
+import {Error} from './pages';
+import { Layout } from './pages/Layout';
+
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <Layout/>,
+    errorElement: <Error />,
+    children: [
+      {
+        index: true,
```

Comment:
Do we need this?
```suggestion
```

Reply:
yes, since this property specifies the main page when rendering